multicore_runtime/cloud_logging_test.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import json
+import logging
+import mock
+import os
+import unittest
+
+import cloud_logging
```

Comment:
So I think the best arguments were towards using absolute imports, since that "nails down" the package name (as it is supposed to be imported by its users) and prevents problems with importing the same module twice under different names. Furthermore the case of running test case on the same directory seem to have been refuted, since the relative import requires a package, so running the test case's .py as a standalone would fail to import the main code module correctly...

*Having said that*, I think to make absolute paths work in a development environment (from a git checkout), you might need to rearrange your tree, so that you have the whole package structure in the repo (so that you have a place where to point `$PYTHONPATH` and import the modules with absolute names), so perhaps you'll want to start with relative imports and do a separate change to update the git directory structure so that absolute imports will work? I'd be willing to accept that (but please take the advice for absolute names in mind and aim towards implementing it!)